lib/b2b/sessions.ts

```diff
@@ -529,11 +544,27 @@ export class Sessions {
     );
 
     const organizationClaim = "https://stytch.com/organization";
-    const { [organizationClaim]: orgClaimUntyped, ...claims } =
-      sess.custom_claims;
+    const rolesClaim = "https://stytch.com/roles";
+    const {
+      [organizationClaim]: orgClaimUntyped,
+      [rolesClaim]: rolesCaimUntyped,
+      ...claims
+    } = sess.custom_claims;
 
     const orgClaim = orgClaimUntyped as { organization_id: string };
+    const roleClaim = rolesCaimUntyped as string[];
+
+    if (params.authorization_check) {
+      const policy = await this.rbacCache.getPolicy();
+      await performAuthorizationCheck({
+        policy,
+        subjectRoles: roleClaim,
+        subjectOrgID: orgClaim.organization_id,
+        authzRequest: params.authorization_check,
+      });
+    }
 
+    // TODO: The Roles claim will probably make its way into here...
```

Comment:
roles will just go into the return statement right? since it's part of the member session proto now